index.html

```diff
@@ -247,19 +288,53 @@ <h1 class="brand-heading">
     </div>
 </section>
 
-<!-- Download Section
-<section id="download" class="content-section text-center">
-    <div class="download-section">
-        <div class="container">
-            <div class="col-lg-8 col-lg-offset-2">
-                <h2>Download Grayscale</h2>
-                <p>You can download Grayscale for free on the preview page at Start Bootstrap.</p>
-                <a href="http://startbootstrap.com/template-overviews/grayscale/" class="btn btn-default btn-lg">Visit Download Page</a>
-            </div>
+<!-- Gallery -->
+<section id="gallery" class="container text-center">
+
+
+    <div class="grid about__pictures">
+        <h3>Gallery</h3>
+
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive fancybox" src="img/mop_screen_1.png" alt="MOP screenshot">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive fancybox" src="img/mop_screen_2.png" alt="MOP screenshot">
         </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive fancybox" src="img/mop_screen_3.jpeg" alt="MOP screenshot">
+        </div>
+        <!-- TODO use javascript to load all images together -->
+        <!--
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>-->
```

Comment:
its not unnecessary per se, its basically boilerplate code for additional images